### ISSUE TITLE
INFRA12 - Define a budget for Accessibility score in Lighthouse check

### ISSUE THREAD

**Issue by gregoriopalama** [gdgpescara/hedwig#121]
Update the Lighthouse check. It should pass only if the budget has been reached. 
Define a budget for the Accessibility score. It should be a reasonable score, chosen to achieve business objectives but to not require too much effort during the development.

**Comment by gregoriopalama** [comment]
We already have axe that performs a check. [Lighthouse's checks](https://developer.chrome.com/docs/lighthouse/accessibility/scoring) are very similar. For this reason, I would stick to a 100% on Accessibility. Going define the budget with a PR
